stock_picking_batch_extended/tests/test_batch.py

```diff
@@ -10,9 +11,9 @@ def _create_product(self, name, product_type="consu"):
         return self.env["product.product"].create({"name": name, "type": product_type})
 
     def setUp(self):
```

Comment:
@BorisFL Could you change this to classmethod ?

Reply:
I have converted it to a classmethod